.github/workflows/run-prerelease.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Activiti Cloud
+
+on:
+  push:
+    branches: ['releases/beta/**']
+
+jobs:
+  load-release-info:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.load-descriptor.outputs.version }}
+      activiti-tag: ${{ steps.load-descriptor.outputs.activiti-tag }}
+      activiti-cloud-tag: ${{ steps.load-descriptor.outputs.activiti-cloud-tag }}
+      activiti-cloud-application-tag: ${{ steps.load-descriptor.outputs.activiti-cloud-application-tag }}
+
+    steps:
+      - name: Installing activiti-scripts
+        uses: actions/checkout@v3
+
+      - uses: Alfresco/alfresco-build-tools/.github/actions/load-release-descriptor@v1.15.0
+        id: load-descriptor
+        with:
+          release-descriptor: release.yaml
+
+  create-git-tags:
+    runs-on: ubuntu-latest
+    needs: load-release-info
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - repo: Activiti
+            base-tag: ${{ needs.load-release-info.outputs.activiti-tag }}
+          - repo: activiti-cloud
+            base-tag: ${{ needs.load-release-info.outputs.activiti-cloud-tag }}
+          - repo: activiti-cloud-application
+            base-tag: ${{ needs.load-release-info.outputs.activiti-cloud-application-tag }}
+    env:
+      REPO_DIR: repos/${{ matrix.repo }}
+    steps:
+      - name: Checkout ${{ matrix.repo }}
+        uses: actions/checkout@v3
+        with:
+          repository: Activiti/${{ matrix.repo }}
+          path: ${{ env.REPO_DIR }}
+          ref: ${{ matrix.base-tag }}
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
```

Comment:
this should be an input

Reply:
This is not a reusable workflow.

Reply:
please disregard: not a reusable workflow...